include/reactor-uc/scheduler.h

```diff
@@ -73,7 +63,22 @@ struct Scheduler {
   void (*register_for_cleanup)(Scheduler *self, Trigger *trigger);
 
   void (*acquire_and_schedule_start_tag)(Scheduler *self);
+
+  // void (*set_duration)(Scheduler *self, interval_t duration);
+
+  lf_ret_t (*add_to_reaction_queue)(Scheduler *self, Reaction *reaction);
```

Comment:
Maybe rename because it isnt necessarily a queue anymore, what about `trigger_reaction`?